source/extensions/config_subscription/grpc/grpc_mux_impl.cc

```diff
@@ -412,6 +419,19 @@ void GrpcMuxImpl::processDiscoveryResources(const std::vector<DecodedResourcePtr
     // updates in the message for EDS/RDS.
     if (!found_resources.empty()) {
       watch->callbacks_.onConfigUpdate(found_resources, version_info);
+      // Resource cache is only used for EDS resources.
```

Comment:
As discussed offline, it would be good to document somewhere (maybe an issue) the strategy we're taking around new / old gRPC mux impl. It's totally fine to do this particular PR for both based on the plan you have, but it would be good to make this visible to others working in this space what is going on and telegraph the old / new gRPC mux behavioral change and migration story.